### ISSUE TITLE
Style multiline with + is indented

### ISSUE THREAD

**Issue by sufleR** [bbatsov/rubocop#1628]
cop is correcting my code in new line after ```+``` is on the end
```ruby
def method_name
  first_long_line_of_code +
    second_long_line_of_code +
    third_long_line_of_code 
end
```

expected:

```ruby
def method_name
  first_long_line_of_code +
  second_long_line_of_code +
  third_long_line_of_code 
end
```
is there configuration option for this ?

Log is reporting
```
: [Corrected] Use 2 (not 0) spaces for indenting an expression spanning multiple lines.
```

**Comment by jonas054** [comment]
No. The style `aligned` (which is the default for `Style/MultilineOperationIndentation`) only enforces aligned operands when the first operand is preceded by something (a keyword such as `if` or an assignment to a variable). The reason why we want to indent the continuation of an operation like the one in your example is that we want to make clear that it's not the start of a a new expression. I find your preferred style a bit strange, almost misleading.

**Comment by sufleR** [comment]
ok I never looked at it this way. thank you for explanation.

exmaple above is not the best one.
Consider such situation:
```
def method_name
  calculate_result_for( 1,      2,   34, 5, 6) +
  calculate_result_for(23, 234, 546, 4, 5) +
  calculate_result_for( 3,      4, 111, 1, 9)
end
```

Is there any posiibility turn it off for a single method ?

**Comment by jonas054** [comment]
Absolutely! Just disable the cop locally.

```ruby
# rubocop:disable Style/MultilineOperationIndentation

def method_name
  calculate_result_for( 1,   2,  34, 5, 6) +
  calculate_result_for(23, 234, 546, 4, 5) +
  calculate_result_for( 3,   4, 111, 1, 9)
end

# rubocop:enable Style/MultilineOperationIndentation
```

Another idea for how to get out of the problem is this:

```ruby
def method_name
  result = calculate_result_for( 1,   2,  34, 5, 6) +
           calculate_result_for(23, 234, 546, 4, 5) +
           calculate_result_for( 3,   4, 111, 1, 9)
  result
end
```

**Comment by sufleR** [comment]
thank you very much.
